app/controllers/admissions_admin/applicants_controller.rb

```diff
@@ -12,4 +12,69 @@ def show_unflagged_applicants
     @admission = Admission.find(params[:admission_id])
     @applicants = Applicant.unflagged_applicants(@admission)
   end
+
+  def show_unlogged_applicants
+    @admission = Admission.find(params[:admission_id])
+    @group_jobs = get_groups_and_unlogged_applicants(@admission)
+    @log_entries = LogEntry.possible_log_entries
+    @applicants = Applicant.unlogged_applicants(@admission)
+
+    respond_to do |format|
+      format.html
+      format.csv
+    end
+  end
+
+  def log_all_applicants
+    admission = Admission.find(params[:admission_id])
+    group_jobs = get_groups_and_unlogged_applicants(admission)
+
+    group_jobs.each do |group, job_applicants|
+      job_applicants.each do |j|
+        puts j.applicant.full_name
+        add_entry_to_applicant(admission, j.applicant, group, params[:log_entry][:log])
```

Comment:
This will create the same `LogEntry` for every applicant, will optimize this when I have the time.